grid2op/Opponent/NeuripsOpponent.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2019-2020, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+import warnings
+import numpy as np
+
+from grid2op.Opponent import BaseOpponent
+from grid2op.Exceptions import OpponentError
+
+
+class NeuripsOpponent(BaseOpponent):
+    def __init__(self, action_space):
+        BaseOpponent.__init__(self, action_space)
+        self._do_nothing = None
+        self._attacks = None
+        self._lines_ids = None
+        self._next_attack_time = None
+        self._attack_cooldown = 12*24 # attack cooldown
+
+        # this is the constructor:
+        # it should have the exact same signature as here
+
+    def init(self, lines_attacked=[], **kwargs):
```

Comment:
in kwargs, add the attack_cooldown if you want.

Reply:
done